frontend/src/hooks/useDefaultProjectSettings.ts

```diff
@@ -39,24 +35,12 @@ export const useDefaultProjectSettings = (
         defaultStickiness,
         refetch,
         loading: !error && !data,
-        status: data?.status,
         error,
     };
 };
 
-export const fetcher = async (path: string): Promise<IStickinessResponse> => {
-    const res = await fetch(path);
-
-    if (res.status === 404) {
-        return { status: 404 };
-    }
-
-    if (!res.ok) {
-        await handleErrorResponses('Project stickiness data')(res);
-    }
-
-    return {
-        status: res.status,
-        body: await res.json(),
-    };
+const fetcher = (path: string) => {
```

Comment:
Why was this refactoring necessary?

Reply:
to match code of the other fetchers I've seen. We don't expose the status normally and we also keep fetcher private to the hook

Reply:
you can check useChangeRequest for comparison